src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -280,6 +270,20 @@ export const PublishingAccordion = ({
             </LoadingButton>
           </Box>
         )}
+
+        {hasPendingTicket && (
+          <Accordion elevation={3} sx={{ maxWidth: '60rem', my: '1rem' }}>
```

Comment:
Siden denne nå er nederst
```suggestion
          <Accordion elevation={3} sx={{ maxWidth: '60rem', mt: '1rem' }}>
```